options.cpp

```diff
@@ -385,8 +387,9 @@ options_t::options_t(int argc, char *argv[]): options_t()
             expire_tiles_zoom_min =
                 static_cast<uint32_t>(std::strtoul(optarg, &next_char, 10));
             if (expire_tiles_zoom_min == 0) {
-                throw std::runtime_error(
-                    "Missing zoom level for tile expiry.\n");
+                throw std::runtime_error("Missing zoom level for tile expiry. "
+                                         "Minimum zoom level must be larger "
+                                         "than 0.\n");
```

Comment:
I'd remove the 'Missing zoom level' part entirely or rename it to 'Bad argument for...'. As is, it seems misleading given that we have checked a couple of lines above that there is an optarg.